api/v1alpha1/openapi.yaml

```diff
@@ -2539,10 +2539,42 @@ components:
         - type: object
           properties:
             inline:
-              type: object
-              additionalProperties: true
+              type: array
+              items:
+                $ref: '#/components/schemas/FileSpec'
           required:
           - inline
+    FileSpec:
+      type: object
+      properties:
+        path:
+          type: string
+          description: The absolute path to the file on the device. Note that any existing file will be overwritten.
+        contents:
```

Comment:
`content` (singular)?

Reply:
I agree with you, but copied from ignition.  Will change.

Reply:
Fixed